### ISSUE TITLE
Caffe model

### ISSUE THREAD

**Issue by ShervinAr** [BichenWuUCB/squeezeDet#41]
Hello,

Is there any chance you could provide your model's structure in Caffe?

many thanks

**Comment by Timen** [comment]
Check out this git repository:

https://github.com/DeepScale/SqueezeNet

Squeezedet builds upon this network by adding fire 10, fire 11 and conv12 and structure for the loss function.  So that might be a good starting point.

**Comment by ShervinAr** [comment]
@Timen  many thanks for your response. could you please provide the conv12 structure in Caffe?

**Comment by Timen** [comment]
I do not have experience with Caffe, so I can't help you with that.

**Comment by BichenWuUCB** [comment]
@ShervinAr Thanks for your question. 

Currently, I don't have a plan to re-implement SqueezeDet in Caffe or other deep learning tools. But if you only need inference or deployment, it should be straightforward to convert SqueezeDet's network structure to protobuf and convert tensorflow checkpoint file to caffemodel file. If anyone plan to implement this conversion, please let me know and I'm happy to add a pointer on the front page to your work.

**Comment by ShervinAr** [comment]
@BichenWuUCB Many thanks for your response.
Yes, actually I only need inference (deployment). Could you please provide some _**hints**_ on how to do such a conversion (i.e. convert SqueezeDet's structure to protobuf and convert the tensorflow check point file to a caffemodel file)?

**Comment by ShervinAr** [comment]
@BichenWuUCB I would also be very thankful if you could let me know the dimensionalities of the outputs of fire10 and fire11 modules in the SqueezDet

**Comment by BichenWuUCB** [comment]
You can find the model definition from [here](https://github.com/BichenWuUCB/squeezeDet/blob/master/src/nets/squeezeDet.py)
